Makefile

```diff
@@ -188,7 +190,7 @@ release-server-xgo: check-xgo ## Create server binaries for release using xgo
 
 release-server: ## Create server binaries for release
 	# compile
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=1 go build -ldflags '${LDFLAGS}' -o dist/server/linux_amd64/woodpecker-server github.com/woodpecker-ci/woodpecker/cmd/server
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o dist/server/linux_amd64/woodpecker-server github.com/woodpecker-ci/woodpecker/cmd/server
```

Comment:
Well, right now, it's m1 or SQLite. 

If you can find a better way to get m1 and SQLite, I'm all for it. And if you need help testing a change to see if it works on m1, I'm happy to test.

Reply:
I'll check to see if this is necessary, I had assumed they were equivalent, but this appears to be a cross compilation from the perspective of an m1.

Note: if you need SQLite, then your test coverage is incomplete because this PR let me run all the tests successfully.

Reply:
You have to set a specific env var to test against one of the database adapters. So you can run tests against sqlite / postgres / mysql.

Reply:
Should we maybe have a look at [https://pkg.go.dev/modernc.org/sqlite?utm_source=godoc]?